ridgeback_base/include/ridgeback_base/ridgeback_passive_joint.h

```diff
@@ -0,0 +1,70 @@
+/**
+ *
+ *  \file
+ *  \brief      Class publishing the joint state for passive front axle.
+ *  \author     Mike Purvis <mpurvis@clearpathrobotics.com>
+ *  \copyright  Copyright (c) 2015, Clearpath Robotics, Inc.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of Clearpath Robotics, Inc. nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL CLEARPATH ROBOTICS, INC. BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * Please send comments, questions, or patches to code@clearpathrobotics.com
+ *
+ */
+
+#ifndef RIDGEBACK_BASE_RIDGEBACK_PASSIVE_JOINT_H
+#define RIDGEBACK_BASE_RIDGEBACK_PASSIVE_JOINT_H
+
+#include "ros/ros.h"
+#include "sensor_msgs/JointState.h"
+
+namespace ridgeback_base
+{
+
+  class PassiveJointPublisher
+  {
+  public:
+    PassiveJointPublisher(ros::NodeHandle& nh)
+    {
+      msg_.name.push_back("front_rocker");
```

Comment:
I know it's goofy, but please pass the joint name and period in to the constructor. Name could even be a `ros::V_string`, though it's hard to imagine any additional passive joints being added to Ridgeback.